static/sass/styles.scss

```
@@ -852,4 +852,10 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
   
   margin-left: 0;
   padding-left: 1rem;
+}
+
+.u-realign-logos {
+  @media screen and (max-width: $breakpoint-large - 1) {
```

Using range in media query would be safer (to avoid potential subpixel issues).
```suggestion
  @media screen and (width < $breakpoint-large) {
```